lib/src/token_storage.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:dart_jsonwebtoken/dart_jsonwebtoken.dart';
+import 'package:doordash_drive_api/doordash_drive_api.dart';
+import 'package:doordash_drive_api/src/exceptions/exceptions.dart';
+
+/// Holds the implementation for authenticating with a Json Web Token
+class TokenStorage {
+  TokenStorage({
+    required AccessKey accessKey,
+  }) : _accessKey = accessKey {
+    _generateToken();
+  }
+
+  late String _token;
+  final AccessKey _accessKey;
+
+  String get getToken {
```

Comment:
```suggestion
  String get token {
```
since this is already a getter, can we rename this?